prepare_inventory.py

```diff
@@ -0,0 +1,47 @@
+#!/bin/env python3
+
+# This is simple script which converts EDPM inventory from the OpenShift's secret
+# to the very simple inventory file which can be used by the RHOSO break/fix training
+# scripts while preparing training scenarios.
+# Usage of the script:
+# python3 prepare_inventory.py <path_to_the_edpm_inventory.yaml> <path_to_the_output_file.yaml>
+#
+# Edpm inventory file can be get from the OpenShift with command like:
+#
+# oc get secret <secret_name> -o json | jq '.data | map_values(@base64d)' | jq -r '.inventory'
+
+
+import yaml
+import sys
+
+
+if __name__ == '__main__':
+    src_file = sys.argv[1]
+    dst_file = sys.argv[2]
+    if not src_file:
+        print("Yaml file with the RHOSO edpm inventory is required "
+              "to be passed as first argument.")
+        sys.exit(1)
+    if not dst_file:
+        print("Output file is required to be passed as second argument.")
+        sys.exit(1)
+
+    with open(src_file) as src:
+        edpm_inventory = yaml.safe_load(src)
+
+    dest_inventory = {}
+    for group_name, group in edpm_inventory.items():
+        if group_name not in dest_inventory:
+            dest_inventory[group_name] = {'hosts': {}}
+        for hostname, host_options in group.get('hosts').items():
```

Comment:
nit: You can use `setdefault` here avoiding the condition
```
hosts = dest_inventory.setdefault(group_name, {'hosts': {}})
for hostname, host_options in hosts.items():
```

Reply:
done, thx